<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterrecetteDialog</class>
 <widget class="QDialog" name="ajouterrecetteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditTitre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDescription">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableIngredients">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>6</number>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantite</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unite</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnAjouterIngredient">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter ingredient</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableInstructions">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Contenu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Parent</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnAjouterInstruction">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter instruction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnValider">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnnuler">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de la recette</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description de la recette</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
